jdaviz/configs/imviz/tests/test_regions.py

```diff
@@ -168,71 +168,73 @@ def test_regions_annulus_from_load_data(self):
 
     def test_photutils_pixel(self):
         my_aper = CircularAperture((5, 5), r=2)
-        bad_regions = self.imviz.load_regions([my_aper], return_bad_regions=True)
+        bad_regions = self.subset_plugin.import_region([my_aper], return_bad_regions=True)
         assert len(bad_regions) == 0
         self.verify_region_loaded('Subset 1')
         assert len(self.imviz.get_interactive_regions()) == 1
 
     def test_photutils_sky_has_wcs(self):
         sky = SkyCoord(ra=337.5202808, dec=-20.833333059999998, unit='deg')
         my_aper_sky = SkyCircularAperture(sky, 0.5 * u.arcsec)
-        bad_regions = self.imviz.load_regions([my_aper_sky], return_bad_regions=True)
+        bad_regions = self.subset_plugin.import_region([my_aper_sky], return_bad_regions=True)
         assert len(bad_regions) == 0
         self.verify_region_loaded('Subset 1')
         assert len(self.imviz.get_interactive_regions()) == 1
 
 
 class TestLoadRegionsFromFile(BaseRegionHandler):
-
-    def setup_class(self):
+    @pytest.fixture(autouse=True)
+    def setup_class(self, imviz_helper):
         self.region_file = get_pkg_data_filename(
             'data/ds9.fits.reg', package='regions.io.ds9.tests')
         self.arr = np.ones((1024, 1024))
         self.raw_regions = Regions.read(self.region_file, format='ds9')
+        self.imviz = imviz_helper

```

Comment:
This changes the fundamental behavior of the test class. I wanted each method to have a new Imviz instance, but this change makes them all share the same instance. So hardcoded check like "Subset 1" is no longer reliable. Please revert this change.